_security/configuration/yaml.md

```diff
@@ -134,6 +134,20 @@ An authentication cache for the Security plugin exists to help speed up authenti
 plugins.security.cache.ttl_minutes: 60
 ```
 
+### Enabling user access to system indexes
+
+Mapping the `system:admin/<system_index_name>` permission to a user allows that user to modify the system index specified in the permission name. The one exception is the Security plugin's [system index]({{site.url}}{{site.baseurl}}/security/configuration/system-indices/). The `plugins.security.system_indices.additional_control.enabled` setting provides a way for administrators to make this permission available for or hidden from role mapping.
+
+When set to `true`, the feature is enabled and administrators can add the `system:admin/<system_index_name>` permission for a user.
+
+```yml
+plugins.security.system_indices.additional_control.enabled: true
+```
+When set to `false`, the permission is disabled and only admins with an admin certificate can make changes to system indexes. By default, the setting is `true` for a new cluster.
```

Comment:
Be default, the setting is false. Refer here:
https://github.com/opensearch-project/security/blob/d77466418d3dfe551ca1ead20fcb4e6fda88515e/src/main/java/org/opensearch/security/support/ConfigConstants.java#L319